index.html

```diff
@@ -1,12 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather Report</title>
+    <meta charset="UTF-8"/>
```

Comment:
found an error when I sent this through the w3 html validator. feel free to check it out here: [https://validator.w3.org/](https://validator.w3.org)